<launch>
   <node pkg="psaf_steering" type="cmd_vel_to_ackermann_drive.py" name="cmd_vel_to_ackermann_drive" output="screen">
    <param name="twist_cmd_topic" value="/cmd_vel" />
    <param name="ackermann_cmd_topic" value="/carla/ego_vehicle/ackermann_cmd" />
    <param name="frame_id" value="ego_vehicle" />
    <param name="wheelbase" value="1.0" />    
 </node>
</launch> 